assets/nasa/starmap_and_constellation_figures_4k.jpg.import

```diff
@@ -18,9 +18,9 @@ dest_files=["res://.godot/imported/starmap_and_constellation_figures_4k.jpg-4444
 [params]
 
 compress/mode=2
+compress/high_quality=false
```

Comment:
Same here, lets see if high quality makes a difference and make sure mipmaps are off.

Reply:
Set to high-quality and disabled mipmap generation.